Plugin: 小爱课程表

**插件名称：**

小爱课程表

**插件功能：**

解放双手，一键导入小爱课表、订阅课表、查询课表

**PyPI 项目名：**

nonebot-plugin-ai-timetable

**插件 import 包名：**

nonebot_plugin_ai_timetable

**插件项目仓库/主页链接：**

https://github.com/maoxig/nonebot-plugin-ai-timetable

**标签：**

[]

# 📃 商店发布检查结果

> Plugin: 小爱课程表

**⚠️ 在发布检查过程中，我们发现以下问题：**
<pre><code><li>⚠️ 插件加载测试未通过。<details><summary>测试输出</summary>插件 nonebot-plugin-ai-timetable 的信息如下：
    name         : nonebot-plugin-ai-timetable 
     version      : 0.1.0                       
     description  :                             
    
    dependencies
     - httpx &gt;=0.23.0,&lt;0.24.0
     - nonebot-adapter-onebot &gt;=2.1.0,&lt;3.0.0
     - nonebot-plugin-apscheduler &gt;=0.2.0,&lt;0.3.0
     - nonebot-plugin-htmlrender *
     - nonebot2 &gt;=2.0.0rc1,&lt;3.0.0
插件 nonebot_plugin_ai_timetable 加载出错：
03-04 15:44:45 [SUCCESS] nonebot | NoneBot is initializing...
03-04 15:44:45 [INFO] nonebot | Current Env: prod
03-04 15:44:46 [SUCCESS] nonebot | Succeeded to import &quot;nonebot_plugin_htmlrender&quot;
03-04 15:44:46 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_htmlrender&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_ai_timetable&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_ai_timetable/__init__.py&quot;, line 13, in &lt;module&gt;
    require(&#x27;nonebot_plugin_htmlrender&#x27;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 165, in require
    plugin = load_plugin(name)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 155, in load_plugin
    raise RuntimeError(
RuntimeError: Module nonebot_plugin_htmlrender is not loaded as a plugin! Make sure not to import it before loading.
03-04 15:44:46 [ERROR] nonebot | Failed to import &quot;nonebot_plugin_ai_timetable&quot;
Traceback (most recent call last):
  File &quot;/home/runner/work/nonebot2/nonebot2/plugin_test/runner.py&quot;, line 4, in &lt;module&gt;
    valid = load_plugin(&quot;nonebot_plugin_ai_timetable&quot;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 34, in load_plugin
    return manager.load_plugin(module_path)
&gt; File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 141, in load_plugin
    module = importlib.import_module(name)
  File &quot;/usr/lib/python3.10/importlib/__init__.py&quot;, line 126, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1050, in _gcd_import
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1027, in _find_and_load
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 1006, in _find_and_load_unlocked
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 688, in _load_unlocked
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/manager.py&quot;, line 233, in exec_module
    super().exec_module(module)
  File &quot;&lt;frozen importlib._bootstrap_external&gt;&quot;, line 883, in exec_module
  File &quot;&lt;frozen importlib._bootstrap&gt;&quot;, line 241, in _call_with_frames_removed
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot_plugin_ai_timetable/__init__.py&quot;, line 13, in &lt;module&gt;
    require(&#x27;nonebot_plugin_htmlrender&#x27;)
  File &quot;/home/runner/.cache/pypoetry/virtualenvs/plugin-test-rYmcgvOP-py3.10/lib/python3.10/site-packages/nonebot/plugin/load.py&quot;, line 167, in require
    raise RuntimeError(f&#x27;Cannot load plugin &quot;{name}&quot;!&#x27;)
RuntimeError: Cannot load plugin &quot;nonebot_plugin_htmlrender&quot;!</details></li></code></pre>
<details><summary>详情</summary><pre><code><li>✅ 项目 <a href="https://github.com/maoxig/nonebot-plugin-ai-timetable">主页</a> 返回状态码 200。</li><li>✅ 包 <a href="https://pypi.org/project/nonebot-plugin-ai-timetable/">nonebot-plugin-ai-timetable</a> 已发布至 PyPI。</li></code></pre></details>

---

💡 如需修改信息，请直接修改 issue，机器人会自动更新检查结果。
💡 当插件加载测试失败时，请发布新版本后在当前页面下评论任意内容以触发测试。

💪 Powered by [NoneBot2 Publish Bot](https://github.com/nonebot/nonebot2-publish-bot)
<!-- PUBLISH_BOT -->

--- a/website/static/plugins.json
+++ b/website/static/plugins.json
@@ -4622,5 +4622,15 @@
     "homepage": "https://github.com/DMCSWCG/nonebot-plugin-AutoRepeater",
     "tags": [],
     "is_official": false
+  },
+  {
+    "module_name": "nonebot_plugin_ai_timetable",
+    "project_link": "nonebot-plugin-ai-timetable",
+    "name": "小爱课程表",
+    "desc": "解放双手，一键导入小爱课表、订阅课表、查询课表",
+    "author": "maoxig",
+    "homepage": "https://github.com/maoxig/nonebot-plugin-ai-timetable",
+    "tags": [],
+    "is_official": false
   }
 ]

